packages/mdc-button/README.md

```diff
@@ -151,4 +151,4 @@ Mixin | Description
 
 ##### Caveat: Edge and CSS Custom Properties
 
-In browsers that fully support CSS custom properties, the above mixins will work if you pass in a [MDC Theme](mdc-theme) property (e.g. `primary`) as an argument. However, Edge does not fully support CSS custom properties. If you are using the `mdc-button-container-fill-color` mixin, you must pass in an actual color value for support in Edge.
\ No newline at end of file
+In browsers that fully support CSS custom properties, the above mixins will work if you pass in a [MDC Theme](mdc-theme) property (e.g. `primary`) as an argument. However, Edge does not fully support CSS custom properties. If you are using the `mdc-button-container-fill-color` mixin, you must pass in an actual color value for support in Edge.
```

Comment:
I can't tell what changed on this line in this PR... but FYI this will conflict with master again because we fixed the link to the theme package.